src/main/app/forms/models/multiRowForm.ts

```diff
@@ -0,0 +1,43 @@
+import { Serializable } from 'models/serializable'
+import { ItemInMultiRowForm } from 'forms/models/itemInMultiRowForm'
+
+export abstract class MultiRowForm<T extends ItemInMultiRowForm> implements Serializable<MultiRowForm<T>> {
```

Comment:
Serializable doesn't serve any purpose. It's not being explicitly consumed anywhere. To be honest, I think it's best if we stop using it. It doesn't even do what the name suggests.